ElementX/Sources/Application/Navigation/NavigationCoordinators.swift

```diff
@@ -23,7 +23,9 @@ class NavigationSplitCoordinator: CoordinatorProtocol, ObservableObject, CustomS
     fileprivate let placeholderModule: NavigationModule
     
     private var cancellables = Set<AnyCancellable>()
-    
+
+    var sidebarWillChangeCancellable: AnyCancellable?
```

Comment:
Optional: maybe slightly clearer to call this
```suggestion
    var sidebarStackModulesCancellable: AnyCancellable?
```